packages/fxa-settings/src/components/Settings/PageRecoveryPhoneRemove/index.test.tsx

```diff
@@ -67,5 +92,67 @@ describe('PageRecoveryPhoneRemove', () => {
     });
 
     expect(account.removeRecoveryPhone).toBeCalled();
+    expect(mockNavigate).toHaveBeenCalledWith('/settings#security', {
+      replace: true,
+    });
+  });
+
+  it.only('navigates to settings page on cancel', async () => {
+    renderWithRouter(
+      <AppContext.Provider value={mockAppContext({ account })}>
+        <SettingsContext.Provider value={mockSettingsContext()}>
+          <PageRecoveryPhoneRemove />
+        </SettingsContext.Provider>
+      </AppContext.Provider>
+    );
+
+    const user = userEvent.setup();
+    await act(async () => {
+      await user.click(screen.getByRole('link', { name: 'Cancel' }));
+    });
```

Comment:
missing assertion to verify what happens after clicking?

Reply:
I'm not sure how to test this since page navigates with 

```
<Link className="link-blue text-sm" to={SETTINGS_PATH}>
  Cancel
</Link>
```

Reply:
I see, might better suited for a functional test then!